apps/dispatch/src/app/components/simulations/view/view.component.ts

```diff
@@ -299,9 +300,15 @@ export class ViewComponent implements OnInit, OnDestroy {
 
   private initSimulationRun(): void {
     this.Simulation$ = this.simulationService
-      .getSimulation(this.uuid)
+      .getSimulation(this.uuid)      
       .pipe(shareReplay(1));
 
+    this.Simulation$.subscribe((simulation: Simulation): void => {
```

Comment:
The additional subscription here is redundant with the formatted simulation below. We can just add this to the chain below to ensure that we don't pass undefined into the format simulation method, which would also avoid the type assertion problem